examples/json/README.md

```diff
@@ -1,11 +1,8 @@
-# GreeterJson
+# Json
 
-The GreeterJson example illustrates how to use JSON to encode the payloads of IceRPC requests and
+The Json example illustrates how to use JSON to encode the payloads of IceRPC requests and
 responses.
 
-It's a variation of the [Greeter](Greeter) example that uses JSON instead of Slice. See also the
```

Comment:
We should explain what this example is about - a simple Greeter that uses the core IceRPC APIs - and encodes request/response payloads with JSON.